src/Tuner.cpp

```diff
@@ -78,16 +91,10 @@ static void sgemmBatched_ref(const std::vector<net_t>& a,
         auto offset_v = batch * n * k;
         auto offset_m = batch * m * n;
 
-        // Calculates transpose(tranpose(A) * B)
```

Comment:
@Ttl I believe this comment (from the existing sources) is actually incorrect?

I get correct working if I put C = A * B_T, or C = (B * A_T)^T, but that's not the same as C = (A_T * B)^T.

Reply:
@Ttl BTW, the CPU version of the winograd transforms might benefit from using Eigen in the matrix math. I find that it certainly becomes more readable!

Reply:
It's row major vs column major issue. The column major C = A * B^T probably does makes more sense.